src/components/feedback/feedback.css

```diff
@@ -0,0 +1,24 @@
+.feedback-container {
```

Comment:
1. this is performance branch?


2. and i mention to you many times, you cant do this..... you have to add this as inline CSS, if you put into file, this code will be gone...
put into the HTML, 
<style>...</style>